scaleout.yml

```diff
@@ -0,0 +1,309 @@
+---
+- hosts: localhost
+  tasks:
+    - name: load instackenv
+      include_tasks: tasks/load_instackenv.yml
+
+    - name: get intrepeter
+      include_tasks: tasks/get_interpreter.yml
+      vars:
+        hostname: "{{ undercloud_hostname }}"
+        user: "stack"
+
+    - name: add undercloud
```

Comment:
If `cleanup.yml` has not been run after overcloud deployment, won't the undercloud still be present in the inventory from the previous deployment? In that case, it wouldn't have to be added to the inventory again.

Although, if a deployment on another allocation has been run before the scaleout, then this step would be required.

Reply:
scaleout.yml is independent of what allocation has been deployed before. 
It can be run on any allocation that is jetpack deployed. So its good to assume that inventory is absent.
Also I have made a change to pass "undercloud_host" via group_vars/all.yml, as undercloud host need not always be the first node in lab generated instackenv.json